github3/orgs.py

```diff
@@ -444,24 +452,32 @@ def events(self, number=-1, etag=None):
         url = self._build_url('events', base_url=self._api)
         return self._iter(int(number), url, Event, etag=etag)
 
-    def members(self, filter=None, number=-1, etag=None):
+    def members(self, filter=None, role=None, number=-1, etag=None):
         r"""Iterate over members of this organization.
 
         :param str filter: (optional), filter members returned by this method.
             Can be one of: ``"2fa_disabled"``, ``"all",``. Default: ``"all"``.
             Filtering by ``"2fa_disabled"`` is only available for organization
             owners with private repositories.
+        :param str role: (optional), filter members returned by their role.
+            Can be one of: ``"all"``, ``"admin"``, ``"member"``. Default:
+            ``"all"``.
         :param int number: (optional), number of members to return. Default:
             -1 will return all available.
         :param str etag: (optional), ETag from a previous request to the same
             endpoint
         :returns: generator of :class:`User <github3.users.User>`\ s
         """
+        headers = {}
         params = {}
         if filter in set(["2fa_disabled", "all"]):
```

Comment:
I think that's a good idea.  I will add and include within this PR along with tests.